controllers/openstackcluster_controller.go

```diff
@@ -491,7 +496,7 @@ func reconcileNetworkComponents(scope scope.Scope, cluster *clusterv1.Cluster, o
 	}
 
 	if openStackCluster.Spec.NodeCIDR == "" {
-		scope.Logger().V(4).Info("No need to reconcile network, searching network and subnet instead")
+		log.Info("No need to reconcile network, searching network and subnet instead")
```

Comment:
Caution when you're making all these mechanical changes: please don't lose the `.V(n)` bit.

```suggestion
		log.V(4).Info("No need to reconcile network, searching network and subnet instead")
```

Reply:
oh right, thanks.